packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -63,28 +63,34 @@ export interface DatepickerProps extends UIComponentProps, Partial<ICalendarStri
    */
   onDateChange?: ComponentEventHandler<DatepickerProps & { value: Date }>;
 
+  /**
+   * Called on error when changing the date.
+   *
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props and proposed value.
+   */
+  onDateChangeError?: ComponentEventHandler<DatepickerProps & { error: string }>;
+
   /** Text placeholder for the input field. */
   placeholder?: string;
 
   /** Target dates can be also entered through the input field. */
   allowManualInput?: boolean;
 
+  /** The component automatically overrides faulty manual input upon blur. */
+  autoCorrectManualInput?: boolean;
```

Comment:
```suggestion
  fallbackToLastCorrectDateOnBlur?: boolean;
```
?

Reply:
Love your proposals 🥇 Changing to a description that describes the exact behavior.